kapajira/jira/issues.py

```diff
@@ -34,13 +35,18 @@ def get_summary(self):
         """ Get issue summary """
         # prevent jira.exceptions.JIRAError:
         # HTTP 400: "The summary is invalid because it contains newline characters."
-        return self._summary.replace("\n", '')
+        return ("Kap Alert: " + self._summary.replace("\n", ''))[:255]
```

Comment:
We do... I got some errors the summary is over 255 characters, that's why I added the 255 limitation, but this will cause the GUID of instance to be cut